data/json/items/tools.json

```diff
@@ -2114,178 +2114,6 @@
     "max_charges": 200
   },
   {
-    "id": "diamond_broadsword",
-    "type": "TOOL",
-    "category": "weapons",
-    "name": "diamond broadsword",
-    "description": "This broadsword's edge has been coated with razor-sharp diamond-like carbon, making you the deadliest knight in the realm.  Your blademaster would be proud.",
-    "weight": 1100,
-    "volume": 7,
-    "price": 185000,
-    "to_hit": 2,
-    "bashing": 8,
-    "cutting": 44,
-    "material": [ "steel", "diamond" ],
-    "symbol": "/",
-    "color": "cyan",
-    "techniques": "WBLOCK_2",
-    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 16 ] ],
-    "flags": [ "CHOP", "DURABLE_MELEE", "SHEATH_SWORD", "VORPAL" ]
-  },
-  {
-    "id": "diamond_katana",
-    "type": "TOOL",
-    "category": "weapons",
-    "name": "diamond katana",
-    "description": "The edge of this katana has been enhanced with a coating of diamond-like carbon.  It is now stronger and sharper than Muramasa could have ever dreamed.",
-    "weight": 1100,
-    "volume": 8,
-    "price": 155000,
-    "to_hit": 1,
-    "bashing": 4,
-    "cutting": 38,
-    "material": [ "steel", "diamond" ],
-    "symbol": "/",
-    "color": "cyan",
-    "techniques": [ "RAPID", "PRECISE", "WBLOCK_2" ],
-    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 16 ] ],
-    "flags": [ "DURABLE_MELEE", "SLICE", "SHEATH_SWORD", "VORPAL" ]
```

Comment:
It doesn't seem to me, the technique is a beheading strike, only large enough blades would have the flag ( which they did ), although the mechanism of acquisition is a bit bizarre ( train on a jabberwoky, reference to lewis carrol's poem )